OPTIONS.md

```diff
@@ -0,0 +1,46 @@
+# VarTable Options
+
+## (Required)
+**--input-vcf**		input VCF file
+
+*	Do we need to differentiate between NGS_Mapper VCF (every position) versus lofreq VCF (only variants)?*
+
+**--input-bam**		input BAM file
+
+*	Do we assume BAM index file is in the same directory as BAM or should we include it as an option?
+
+*--input-bai	input BAM index file*
+
+**--input-ref**		input reference FASTA file
+
+*	Assuming we are using a GenBank ref, are we planning to parse annotated data?*
```

Comment:
It makes sense to allow a genbank file instead of fasta as input for the reference. Then the annotation can be used by the translation function of vartable